src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -886,6 +891,17 @@ const Packages = () => {
                     onChange={handleSearch}
                   />
                 </InputGroupItem>
+                {searchTerm && (
+                  <InputGroupItem>
+                    <Button
+                      variant="control"
+                      aria-label="clear-package-search"
+                      onClick={handleClear}
+                    >
+                      <TimesIcon />
```

Comment:
The Button component supports `icon` prop, could you use it?

Reply:
Right, I sure could! Fixed